net.runelite.RuneLite.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```

Comment:
You have data files but never install them.

Reply:
Yep, same thing with the appdata file.

Reply:
This was fixed.